protos/build.rs

```diff
@@ -1,7 +1,7 @@
 extern crate protoc_grpcio;
 
 fn main() {
-    let proto_root = "src/";
+    let proto_root = &format!("{}{}", "src", std::path::MAIN_SEPARATOR);
```

Comment:
I'm sure this works, but I think it would be a bit neater using the [`join`](https://doc.rust-lang.org/std/path/struct.Path.html#method.join) approach.

```rust
use std::path::{Path, PathBuf};

assert_eq!(Path::new("/etc").join("passwd"), PathBuf::from("/etc/passwd"));
```

Reply:
I had some trouble using the join approach, but it seems like if you simply create a new Path object "src" it works. It builds and passes all tests for my machine and it passes on travis-ci as well. Have a look and see what you think, maybe it just works™